client/src/components/NotifsDrawer.js

```diff
@@ -0,0 +1,127 @@
+import React, { useState, useEffect } from "react";
+import {
+  Drawer,
+  List,
+  ListItem,
+  ListItemText,
+  Typography,
+  ListItemAvatar,
+  ListItemIcon,
+} from "@material-ui/core";
+import ChatIcon from "@material-ui/icons/Chat";
+import MonetizationOnIcon from "@material-ui/icons/MonetizationOn";
+import { Link } from "react-router-dom";
+import notificationsAPI from "../notificationsAPI";
+
+const NotifsDrawer = ({
+  notifsOpen,
+  setNotifsOpen,
+  classes,
+  setUnreadCount,
+}) => {
+  const [notifs, setNotifs] = useState([]);
+
+  useEffect(() => {
+    setNotifs(notificationsAPI());
+  }, []);
+
+  useEffect(() => {
+    setUnreadCount(notifs.length);
+  }, [notifs, setUnreadCount]);
+
+  const notificationClickHandler = (notif) => {
+    notifs.splice(
+      notifs.findIndex((n) => n._id === notif._id),
+      1
+    );
```

Comment:
If you pass in the index while mapping `notifs`, you could simplify this to:
```suggestion
  const notificationClickHandler = (notif, index) => {
    notifs.splice( index, 1 );
```

Reply:
Good suggestion, thanks!